usaspending_api/recipient/delta_models/recipient_lookup.py

```diff
@@ -33,6 +33,23 @@
     **{k: v for k, v in RPT_RECIPIENT_LOOKUP_DELTA_COLUMNS.items()},
 }
 
+unlinked_recipient_anti_join = """
+    /* Begin left anti joins to ensure we are not loading any recipients
+    that are not linked to transactions */
+    LEFT JOIN raw.published_fabs pf
+    ON {sam_recipient_alias}.uei = pf.uei
+        AND {sam_recipient_alias}.uei = pf.ultimate_parent_uei
+
+    LEFT JOIN raw.detached_award_procurement dap
+    ON {sam_recipient_alias}.uei = dap.awardee_or_recipient_uei
+        AND {sam_recipient_alias}.uei = dap.ultimate_parent_uei
```

Comment:
In a lot of cases, the UEI and Ultimate Parent UEi will not be the same. Makes more sense to use `OR` here : `ON UEI OR Ultimate Parent UEI`

Reply:
Done